projects/observability/src/pages/apis/service-detail/instrumentation/service-instrumentation.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+import { BreadcrumbsService } from '@hypertrace/components';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Component({
+  styleUrls: ['./service-instrumentation.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <main class="service-instrumentation">
+      <section class="overview">
+        <div>{{ this.serviceName$ | async }} progress circle and description</div>
+        <div>org scores</div>
+      </section>
+
+      <section class="checks-container">
+        <div>card1</div>
+        <div>card2</div>
+        <div>card3</div>
+        <div>card4</div>
+      </section>
+    </main>
+  `
+})
+export class ServiceInstrumentationComponent {
```

Comment:
So, verifying that the `resolver` for `/service` route is actually bringing the `serviceName` to `breadCrumbsService` and we are just leveraging the same.

Reply:
Also, what happens when we have navigated to `/instrumentation` tab, but service name isn't yet resolved?

Reply:
Yes, didn't look into how service name is resolved. But figured if it works for page-header, should work for the tab too.

The header uses the breadcrumbService as well:

https://github.com/razorpay/hypertrace-ui/blob/rzp_main/projects/components/src/header/page/page-header.component.ts#L106-L108

<img width="467" alt="Screenshot 2022-09-02 at 12 09 00 PM" src="https://user-images.githubusercontent.com/29686866/188074407-16177600-dfcb-4247-b03a-0fdfd287ad89.png">

Reply:
- Go to your tab.
- Update Duration at top to `1 day`
- Reload the page.
- See when service name is available, just want to ensure it doesn't block your page.

Reply:
Yes, works 👍🏻